osquery/database/plugins/rocksdb.cpp

```diff
@@ -201,7 +208,16 @@ Status RocksDBDatabasePlugin::setUp() {
   // Attempt to create a RocksDB instance and handles.
   auto s =
       rocksdb::DB::Open(options_, path_, column_families_, &handles_, &db_);
+
+  if (s.IsCorruption()) {
+    // The database is corrupt - try to repair it
+    repairDB();
+    s = rocksdb::DB::Open(options_, path_, column_families_, &handles_, &db_);
+  }
+
   if (!s.ok() || db_ == nullptr) {
+    LOG(INFO) << "Rocksdb open failed [" << s.code() << ":" << s.subcode()
```

Comment:
Perhaps:
```cpp
LOG(ERROR) << "Cannot open database: " << path_ << " (" << s.code() << ":" << s.subcode() << ") " << s.ToString();
```

I see we use "[" in some places, but () are more common for error codes.